packages/material-ui/src/FormLabel/FormLabel.js

```diff
@@ -34,6 +34,7 @@ export const styles = theme => ({
   filled: {},
   /* Styles applied to the root element if `required={true}`. */
   required: {},
+  /* Styles applied to the asterisk element if `required={true}`. */
```

Comment:
```suggestion
  /* Styles applied to the asterisk element. */
```
(`required` just controls whether the asterisk is displayed, not whether it receives this style.)

Also, could you move this above the "placeholder" classes? We keep those at the bottom in all the other components. (We also need to come up with a better description for those, but that's for another PR.)

Reply:
Thanks for reviewing. I have added requested changes.